Gemfile

```diff
@@ -10,6 +10,7 @@ gem 'rake'
 gem 'rspec', '~> 3.11'
 gem 'rubocop-performance', '~> 1.7'
 gem 'rubocop-rake', '~> 0.6'
+gem 'simplecov'
```

Comment:
I had some old version of `simplecov` installed, 
```
Warning: Error occurred while trying to load /Users/pirj/source/rubocop-rspec/.simplecov. Error message: undefined method `enable_coverage' for main:Object
```

Updating it resolved the issue:
```
Fetching simplecov 0.22.0 (was 0.17.1)
```

Reply:
I tried to specify the version. What do you think of this?
https://github.com/rubocop/rubocop-rspec/blob/3bd319323f5959a29617941f89fa1550f70c18ec/Gemfile#L13